clients/drcachesim/analyzer_multi.cpp

```diff
@@ -422,7 +422,8 @@ analyzer_multi_tmpl_t<RecordType, ReaderType>::analyzer_multi_tmpl_t()
                 nullptr, op_verbose.get_value(), op_jobs.get_value(),
```

Comment:
> Relaxes raw2trace error reporting to allow some recoverable errors during libipt decode. 

"recoverable" sounds like the error was fixed and the final trace has no problems: but isn't the error still there and the final trace has PC discontinuities?  Please make this clear.  If I saw "10 recoverable errors" I would think "ok, there was something weird, but it was worked around and now my final trace has no problems."  Why would that need to be under an option though?  I believe you're using the term "recoverable" from the viewpoint of getting the PT library to keep going and not die even if it leaves a gap.  But from the end user point of view: the user doesn't care if you had to try N times to get the PT library to work; only whether the final trace is ok.  I would say to focus on that end trace condition.  IMHO this should be described as "allowing discontinuities in kernel traces".  The option name should reflect that and the description/docs should explicitly talk about the gaps; the user has no idea what the consequences of "recoverable errors" are w/o knowing what those errors are and how they were "recovered".

Ditto in several other places.

Reply:
I agree that the "recoverable" term is ambiguous. I replaced the option to say pt2ir_best_effort (or can alternatively change it to pt2ir_ignore_errors; see other comment thread). We do not ignore only discontinuity errors in the trace but also some other (mysterious) decode errors that seem to affect  ~1% of all  syscall traces.